control/control.go

```diff
@@ -282,34 +289,43 @@ func step2(cfg Config) error {
 			}
 		}
 
-		results = make(chan result)
-		requests := make(chan request, cfg.Step2.Clients)
-
 		h, done := newReadHandlers(cfg)
 		if done != nil {
 			defer done()
 		}
-
-		bar = pb.New(cfg.Step2.TotalRequests)
-		bar.Format("Bom !")
-		bar.Start()
-
-		for i := range h {
-			wg.Add(1)
-			go func(rh ReqHandler) {
-				defer wg.Done()
-				doHandler(context.Background(), rh, requests)
-			}(h[i])
+		reqGen := func(reqs chan<- request) { generateReads(cfg, key, reqs) }
+		generateReport(cfg, h, reqGen)
+	case "read-oneshot":
+		key, value := sameKey(cfg.Step2.KeySize), vals.strings[0]
+		logger.Infof("writing key for read-oneshot [key: %q | database: %q]", cfg.Database)
```

Comment:
`logger.Infof("writing key for read-oneshot [key: %q | database: %q]", key, cfg.Database)`? Or delete `key: %q`